pkg/agent/openflow/pipeline.go

```diff
@@ -1493,18 +1507,43 @@ func (c *client) serviceLearnFlow(groupID binding.GroupIDType, svcIP net.IP, svc
 	} else if protocol == binding.ProtocolSCTP {
 		learnFlowBuilder.MatchSCTPDstPort(svcPort)
 		learnFlowBuilderLearnAction = learnFlowBuilderLearnAction.MatchLearnedSCTPDstPort()
+	} else if protocol == binding.ProtocolTCPv6 {
```

Comment:
Use switch-case instead of so many if-else if ?

Reply:
OK